### PR TITLE
feat: add search domain favicon

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
Add favicon for the domains on the search result

### CODE REVIEW

File: src/components/customResults/ResultFavicon.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      setRetryCount((count) => count + 1);
+      return;
+    }
+
+    setSrc(fallbackUrl);
+  }, [fallbackUrl, numbersOfRetry, retryCount]);
+
+  return (
+    <Image
+      key={retryCount.toString()}
```
Comment (Extheoisah) [inline src/components/customResults/ResultFavicon.tsx:29]: This key isn't unique. We can use the src + retryCount + date

**Comment by Extheoisah** [comment]
this favicon is for [bitcoinoptech](https://bitcoinops.org/) not linux mailing list
image

**Comment by BalogunofAfrica** [comment]
> this favicon is for [bitcoinoptech](https://bitcoinops.org/) not linux mailing list image

Did you try the latest preview. That issue was what my last commit aimed to fix https://github.com/bitcoinsearch/bitcoinsearch-app/pull/104/commits/4f5e24a1b236916d98d041064453aefc67563e6d

**Comment by Extheoisah** [comment]
> > this favicon is for [bitcoinoptech](https://bitcoinops.org/) not linux mailing list image
> 
> Did you try the latest preview. That issue was what my last commit aimed to fix [4f5e24a](https://github.com/bitcoinsearch/bitcoinsearch-app/commit/4f5e24a1b236916d98d041064453aefc67563e6d)

You're right. I was testing an old deployment. Its fixed in the latest deployment

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**